test/ContentEncodingHelperTest.cpp

```diff
@@ -0,0 +1,66 @@
+// Copyright 2022, University of Freiburg,
+// Chair of Algorithms and Data Structures.
+// Author: Robin Textor-Falconi (textorr@informatik.uni-freiburg.de)
+
+#include <gtest/gtest.h>
+
+#include "../src/util/HttpServer/ContentEncodingHelper.h"
+#include "../src/util/HttpServer/beast.h"
+
+using namespace ad_utility::content_encoding;
+namespace http = boost::beast::http;
+using ad_utility::content_encoding::CompressionMethod;
+
+TEST(ContentEncodingHelperTest, TestNoneHeaderIsNotSetCorrectly) {

```

Comment:
Remove all occurenences of `Test` in the names, so
`TEST(ContentEncodingHelper, NonHeaderIsSetCorrectly)`
(we want it to be set correctly:))
(similar below).